cli/src/pcluster/config/validators.py

```diff
@@ -1348,21 +1355,21 @@ def ebs_volume_iops_validator(section_key, section_label, pcluster_config):
     section = pcluster_config.get_section(section_key, section_label)
     volume_size = section.get_param_value("volume_size")
     volume_type = section.get_param_value("volume_type")
-    volume_type_to_iops_ratio = {"io1": 50, "io2": 500}
+    volume_size_to_iops_ratio = {"io1": 50, "io2": 500, "gp3": 500}
```

Comment:
I think it is more meaningful to name it `volume_size_to_iops_ratio`. Different volume type has different iops, the ratio is equal to volume size/iops, we also have newly defined `volume_throughput_to_iops_ratio` in the later section, which is only apply on gp3 volume, the ratio is equal to volume_throughput/volume_iops, so I think change the name will make them consistent.

Reply:
ok the reason why I found this confusing is because I was expecting a dict mapping volume_size to iops_ratio while in the dict I see a volume_type being mapped to something

Reply:
Done!